src/importers/category.ts

```diff
@@ -19,9 +19,13 @@ const importCategories = (
       const relationships = category.relationships
       const included = response.included
 
-      const taxonChilds = () => {
-        return relationships.children.data.map((obj) => {
-          const children = included.find((item) => {
+      const taxonChilds = (taxons) => {
+        const sortedRaxons = taxons.data.sort((a, b) => {
+          return Number(a.id) + Number(b.id)
+        })
+
+        return sortedRaxons.map((obj, index) => {
+          const children = included.find(item => {
```

Comment:
Can use `findIncludedOfType` in `src/utils/index.ts` => `findIncludedOfType(item, 'taxons')`